src/nodes/user/jwt.ts

```diff
@@ -0,0 +1,66 @@
+import { getCtx } from "@owl-factory/next";
+import { isClient } from "@owl-factory/utilities/client";
+import { User } from "@prisma/client";
+import { sign, verify } from "jsonwebtoken";
+import { parseCookies, setCookie } from "nookies";
+
+const jwtSecret = process.env.JWT_SECRET;
+const expiresIn = 14 * 24 * 60 * 60;
+const jwtCookieKey = "jwt_reroll_app";
+
+let userCache: User | undefined;
+
+function signUserJwt(user: User) {
```

Comment:
Comment functions, in case the NextAuth doesn't work or happens farther out in the future